ci.sh

```diff
@@ -2,7 +2,7 @@
 set -ex
 
 # Fixed `minirust-tooling` commit, we need to bump this occasionally.
-TOOLING_COMMIT="a0a4a5b"
+TOOLING_COMMIT="7531357"
```

Comment:
This will collide with the intrinsics-followup PR.
I'd prefer this one to go through quicker.